src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
    </div>
```
I think this code looks cleaner and easier to read